observers/screenshot_capture.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
```

Comment:
does this need to be python? now the scheduling is goose, can't it be simple script to take the screenshots vs requiring a py dependency? (thinking of cases where people may run without python- just the recipes but need something to be snapshotting the screen every 20seconds)